src/singletons/WindowManager.cpp

```diff
@@ -195,6 +195,7 @@ void WindowManager::updateWordTypeMask()
     flags.set(settings->animateEmotes ? MEF::BitsAnimated : MEF::BitsStatic);
 
     // badges
+    flags.set(MEF::BadgeSharedChannel);
```

Comment:
should there be a new setting to toggle visibility of this badge? (or shall we lean on an existing setting?)

Reply:
I'm fine to not have a setting for this to begin with